src/Analyzers/Core/Analyzers/Helpers/DiagnosticHelper.cs

```diff
@@ -74,7 +73,7 @@ internal static class DiagnosticHelper
         /// </param>
         /// <param name="tagIndices">
         /// a map of location tag to index in additional locations.
-        /// <see cref="AbstractRemoveUnnecessaryParenthesesDiagnosticAnalyzer{TLanguageKindEnum, TParenthesizedExpressionSyntax}"/> for an example of usage.
+        /// "AbstractRemoveUnnecessaryParenthesesDiagnosticAnalyzer" for an example of usage.
```

Comment:
We can revert this change once this analyzer is moved to the shared analyzers layer.